study.md

```diff
@@ -74,13 +74,13 @@ What are the similarities and differences between Ruby's and JavaScript's
 Similarities:
 
 ```md
-<!-- your answer here -->
+both add the declared arguments to the beginning of an existing array in order listed
 ```
 
 Differences:
 
 ```md
-<!-- your answer here -->
+I could not find any.
```

Comment:
Ruby's unshift array method returns the modified array, allowing method chaining.

JavaScript's unshift array method returns the length of the modified array.